ivy_tests/test_ivy/helpers/assertions.py

```diff
@@ -149,7 +149,7 @@ def value_test(
             )
 
 
-def check_unsupported_dtype(*, fn, input_dtypes, all_as_kwargs_np):
+def check_unsupported_dtype(*, fn, input_dtypes, all_as_kwargs_np, device):
```

Comment:
We'd need a default value for `device` here to avoid failures like [this](https://github.com/unifyai/ivy/actions/runs/5465488875/jobs/9950458070?pr=18263#step:3:135)

Reply:
Should `ivy.default_device` do? Or is `cpu` simply better?

Reply:
tbh it doesn't really matter to the framework, so either would work for me 😄